main.go

```diff
@@ -130,6 +131,8 @@ func main() {
 	flag.StringVar(&introspectBindAddr, "introspect-bind-addr", ":22775",
 		"Port for serving the introspection API")
 	flag.BoolVar(&leaseOnly, "lease-only", false, "Controller uses lease only for leader election")
+	flag.BoolVar(&enableWindowsPrefixDelegation, "enable-windows-prefix-delegation", true,
```

Comment:
Can we make the default to 'false' thus we only need to turn this on for 1.25+

Reply:
Yeah sure, updated it to `false`